app/javascript/components/configureStore.js

```diff
@@ -0,0 +1,22 @@
+import { applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
+
+const GET_MESSAGES_SUCCESS = 'GET_MESSAGES_SUCCESS'
+const initialState = {
+  messages: []
+}
+
+function rootReducer(state, action) {
+  console.log(action.type);
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly keep your codebase clean by making sure to remove all debugging code like `console.log` when you no longer need them.